pytest_answer.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+local-dir=/home/travis/build/yt-project/yt/answer-store
```

Comment:
this obviously needs to be updated (provided the file is even relevant still)

Reply:
@neutrinoceros What's the new path supposed to be?

Reply:
I don't know, but we stopped using Travis a couple mouths ago so it's clearly not usable anymore as it stands.

Reply:
@Xarthisius Do you know the updated path?

Reply:
it's not used right now. We'll change the path in a separate PR that will enable pytest with GHA. Leave it for now.

Reply:
fine, but can we at least change the value to a meaning placeholder like "UNSET" ?

Reply:
same as above:
this file is now gone right ? In which case, you can resolve this @jcoughlin11